src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/WorkloadsTest.java

```diff
@@ -1236,5 +1236,91 @@ public void testCisco_CallManager() {
         Assert.assertTrue(report.getSsaEnabled());
     }
 
+    @Test
+    public void testOracleJDK8OnLinux() {
+        testOracleJDK("JAVA_VERSION=\"1.8.11", "Workloads_Oracle_JDK_8_On_Linux", "Oracle JDK 8");
+
+    }
+
+    @Test
+    public void testOracleJDK11OnLinux() {
+        testOracleJDK("JAVA_VERSION=\"11", "Workloads_Oracle_JDK_11_On_Linux", "Oracle JDK 11");
+
+    }
+
+    @Test
+    public void testOracleJDK13OnLinux() {
+        testOracleJDK("JAVA_VERSION=\"13", "Workloads_Oracle_JDK_13_On_Linux", "Oracle JDK 13");
+
+    }
+
+    private void testOracleJDK(String versionCheckText, String ruleNameCheckText, String workloadCheckText)
+    {
+        Map<String, Object> facts = new HashMap<>();
+        // always add a String fact with the name of the agenda group defined in the DRL file (e.g. "SourceCosts")
+        facts.put("agendaGroup", "Workloads");
+
+        VMWorkloadInventoryModel vmWorkloadInventoryModel = new VMWorkloadInventoryModel();
+        Map<String, String> files = new HashMap<>();
+        files.put("/usr/java/latest/release", versionCheckText);
+        vmWorkloadInventoryModel.setFiles(files);
+        List<String> systemServicesNames = new ArrayList<>();
+        systemServicesNames.add("NOTwas");
+        vmWorkloadInventoryModel.setSystemServicesNames(systemServicesNames);
+        facts.put("vmWorkloadInventoryModel", vmWorkloadInventoryModel);
+
+        WorkloadInventoryReportModel workloadInventoryReportModel = new WorkloadInventoryReportModel();
+
+        facts.put("workloadInventoryReportModel",workloadInventoryReportModel);
+
+        Map<String, Object> results = createAndExecuteCommandsAndGetResults(facts);
+
+        Assert.assertEquals(3, results.get(NUMBER_OF_FIRED_RULE_KEY));
+        Utils.verifyRulesFiredNames(this.agendaEventListener, "AgendaFocusForTest", "SsaEnabled_System_Services_Present", ruleNameCheckText);
```

Comment:
What do you think moving : 
`Utils.verifyRulesFiredNames(this.agendaEventListener, "AgendaFocusForTest", "SsaEnabled_System_Services_Present", ruleNameCheckText);`
to
```
String[] rulesNamesX = {"AgendaFocusForTest", "SsaEnabled_System_Services_Present", ruleNameCheckText};
Utils.verifyRulesFiredNames(this.agendaEventListener,Arrays.stream(rulesNamesX).filter(e-> e != null).toArray(String[]::new));
```
In order to allow receiving the parameter "null"